doc/changelog/02-specification-language/13106-doc-and-changelog-for-13106.rst

```diff
@@ -0,0 +1,5 @@
+- **Removed:**
+  Undocumented and experimental forward class hint feature ``:>>``.
+  Use ``:>`` (see :n:`@of_type`) instead
+  (`#13106 <https://github.com/coq/coq/pull/13106>`_,
+  by Pierre-Marie Pédrot).
```

Comment:
@Zimmi48, shouldn't your name appear here?  And the PR number should be 13106?

Reply:
That's right!